packages/components/src/systems/ScreeningAlerts/ScreeningAlertAcknowledgement.tsx

```diff
@@ -38,6 +39,31 @@ export function ScreeningAlertAcknowledgementDialog(
   props: ScreeningAlertAcknowledgementDialogProps
 ) {
   const [isOpen, setIsOpen] = createSignal<boolean>(true);
+  let ref: Ref<any> | undefined;
+
+  function ignoreWarningAndCreateAnyway() {
+    dispatchDatadogAction(
+      'prescribe-clinical-alerting-acknowledge-alerts',
+      props.alerts.reduce((acc, value, index) => {
+        acc[index] = value.description;
+        return acc;
+      }, {} as Record<string, string>),
+      ref
+    );
+    props.onIgnoreWarningAndCreateAnyway();
+  }
+
+  function onRevisitPrescriptions() {
```

Comment:
here's an event that'll show us which warnigns are encouraging people to cancel creation and modify the prescriptions